examples/src/main/scala/za/co/absa/spline/example/dce/MyOtherJobC.scala

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2020 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.spline.example.dce
+
+import za.co.absa.spline.SparkApp
+import za.co.absa.spline.harvester.SparkLineageInitializer.SparkSessionWrapper
+
+object MyOtherJobC extends SparkApp("Other Job C") {
+
+  spark.enableLineageTracking()
+
+  spark.read
+    .csv("data/output/dce/publish/Transactions/2020/01/01/v1")
+    .write
+    .mode("overwrite")
+    .save("data/_temp/my-other-job-output")
+
+}
```

Comment:
Could the code duplication be refactored away? 
Finding out if the Jobs are really identical takes some effort. If they used the same function (for example) it would be much easier to be sure.

Reply:
I tend to be conservative with code reuse in tests and examples. They are made primarily for reading and copy/pasting by users that are looking at the project for the 1st time. Any loop, inheritance or even a helper method call would not contribute to readability in comparison with a few-line of flat plain and pure Spark code snippet. I would even inline "extends SparkApp", but that's OK. As long as you can copy/past it into the spark-shell and it works, it's fine.

Reply:
As a 1st time reader I had a problem to understand whether MyOtherJobs are very similar or identical. So that doesn't contribute to readability either.

I agree that it makes sense to keep things simple. But these examples make sense only when run together. And therefore I have to understand what they are supposed to do together and for that It really helps to know that the code is identical. Which is easy to see when there is just one code.

Alternatively there could be just one Job and you could just run it three times.